.github/workflows/build_linux_packages.yml

```diff
@@ -123,28 +123,28 @@ jobs:
             --output build/indexer.py
           python build/indexer.py -f '*.tar.xz*' build/artifacts/
           python build/indexer.py -f '*.log' build/logs/
-          sed -i 's,a href="..",a href="../../index-${{env.AMDGPU_FAMILIES}}.html",g' build/logs/index.html
+          sed -i 's,a href="..",a href="../../index-${{env.AMDGPU_FAMILIES}}-linux.html",g' build/logs/index.html
 
       # TODO: Move to script
       - name: Upload Artifacts
         run: |
-          aws s3 cp build/artifacts/ s3://therock-artifacts/${{github.run_id}}/ \
+          aws s3 cp build/artifacts/ s3://therock-artifacts/${{github.run_id}}-linux/ \
             --recursive --no-follow-symlinks \
             --exclude "*" \
             --include "*.tar.xz*"
-          aws s3 cp build/artifacts/index.html s3://therock-artifacts/${{github.run_id}}/index-${{env.AMDGPU_FAMILIES}}.html
+          aws s3 cp build/artifacts/index.html s3://therock-artifacts/${{github.run_id}}-linux/index-${{env.AMDGPU_FAMILIES}}-linux.html
```

Comment:
If the OS name is already in the path, I think it does not need to be added to the filename of the index file?